packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+
+Nigel chat attachment chip:
+
+![](./spec-images/Nigel-attachment-chip.PNG)
+
+[Infragistics Chip component](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/inputs/chip)
+
+---
+
+## Design
+
+### API
+
+_The key elements of the component's public API surface:_
+
+- _Component Name_: `nimble-chip`
+- _Props/Attrs_:
+    - `removable` - set to show the remove button
+    - `appearance` - supports `outline` and `block` appearances
+    - `disabled` - styles the chip in the typical Nimble manner, including any slotted content, and prevents all interaction with it including being able to focus it
+- _Methods_
+- _Events_
+    - `remove` - fired when the chip remove button is pressed (or \<Del\> or \<Backspace\> is pressed when the chip has focus), and `removable` has been set.
+- _Slots_
+    - `start` - content placed to the left of the chip text
+    - (default) - for the primary label text
+- _CSS Classes and CSS Custom Properties that affect the component_
+    - not supported
+
+### Examples
+
+```html
+<nimble-chip> "Option Label" </nimble-chip>
+```
+
+![](./spec-images/Chip-normal.PNG)
+
+```html
+<nimble-chip prevent-remove> "Option Label" </nimble-chip>
+```
+
+![](./spec-images/Chip-no-remove.PNG)
+
+```html
+<nimble-chip>
+    <nimble-icon-check slot="start"> </nimble-icon-check>
+    "Option Label"
+</nimble-chip>
+```
+
+![](./spec-images/Chip-checked.PNG)
+
+### Anatomy
+
+```html
+<template aria-disabled="${x => x.disabled}">
+    <div class="control" part="control">
+        ${startSlotTemplate(context, definition)}
+        <span class="content" part="content">
+            <slot></slot>
+        </span>
+        ${endSlotTemplate(context, definition)}
+        ${when(x => !x.preventRemove, html<Chip>`
+            <nimble-button content-hidden @click="${x => x.handleRemoveClick()}" aria-label="${x => x.removeButtonLabel}">
+                <nimble-icon-times slot="start">
+                </nimble-icon-times>
+                Remove
+            </nimble-button>
+        `)}
+    </div>
+</template>
+```
+
+While the template will have a named `end` slot, this will not be documented following existing Nimble patterns. We may consider using the `end` slot if/when we add an error state to the chip.
+
+### Angular integration
+
+An Angular wrapper will be created for `nimble-chip`. No `ControlValueAccessor` is needed.
+
+### Blazor integration
+
+Blazor integration for the `nimble-chip` will be provided.
+
+### Visual Appearance
+
+See Figma document link at the top of this document.
+
+---
+
+## Implementation
+
+### States
+
+The CSS `hover` and `focusVisible` states will have specific stylings (as seen in the UX designs), in addition to styling capturing the `disabled` attribute state.
+
+### Accessibility
+
+_Consider the accessibility of the component, including:_
+
+- _Keyboard Navigation and Focus_
+    - the chip component is focusable, as is the remove button (when present).
```

Comment:
Why does the chip need to be focusable? Would it make more sense for the chip not to be focusable (and not have keyboard interactions associated with it) and just have the remove button be focusable? That would be more aligned with the nimble-banner.